src/PubSub/Snapshot.php

```diff
@@ -137,6 +137,8 @@ public function info()
      * @param array $options [optional] Configuration options.
      * @return array
      * @throws \BadMethodCallException If a subscription name cannot be found.
+     * @experimental
+     * @whitelisted
```

Comment:
I have added these annotations - should this whole class be annotated in a similar way? And should the whole class be described as "whitelisted", or perhaps as potentially unavailable?